Examples/ObjectiveC/LiveDataExample.m

```diff
@@ -0,0 +1,51 @@
+
+#import "LiveDataExample.h"
+@import Mapbox;
+
+NSString *const MBXExampleLiveData = @"LiveDataExample";
+
+@interface LiveDataExample () <MGLMapViewDelegate>
+
+@property (nonatomic, strong, nullable) NSTimer *timer;
+@property MGLShapeSource *source;
+@end
+
+@implementation LiveDataExample
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    
+    MGLMapView *mapView = [[MGLMapView alloc] initWithFrame:self.view.bounds styleURL:[MGLStyle darkStyleURL]];
+    mapView.autoresizingMask = UIViewAutoresizingFlexibleHeight | UIViewAutoresizingFlexibleWidth;
+    mapView.delegate = self;
+    
+    mapView.tintColor = [UIColor darkGrayColor];
+    [self.view addSubview:mapView];
+}
+
+- (void)mapView:(MGLMapView *)mapView didFinishLoadingStyle:(MGLStyle *)style {
+    // Add a source to the map. https://wanderdrone.appspot.com/ generates coordinates for simulated paths.
+    NSURL *url = [NSURL URLWithString:@"https://wanderdrone.appspot.com/"];
+    _source = [[MGLShapeSource alloc] initWithIdentifier:@"drone-source" URL:url options:nil];
```

Comment:
The source identifier doesn’t need to contain “source”; it can be `wanderdrone` or simply `drones`.